lucille-core/src/main/java/com/kmwllc/lucille/stage/FetchUri.java

```diff
@@ -0,0 +1,122 @@
+package com.kmwllc.lucille.stage;
+
+import com.kmwllc.lucille.core.ConfigUtils;
+import com.kmwllc.lucille.core.Document;
+import com.kmwllc.lucille.core.Stage;
+import com.kmwllc.lucille.core.StageException;
+import com.typesafe.config.Config;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Iterator;
+import nl.altindag.ssl.util.UriUtils;
```

Comment:
Do we need this import?

Reply:
Not required, updated this in new commit